### DISCUSSION TITLE
Problem about over fit

### DISCUSSION THREAD

**Discussion by thevelvetunderground** [lululxvi/deepxde#1249] [Q&A]
Dear Pro. Lu
I use PINN to solve a Laplace's equation：
<img width="277" alt="image" src="https://user-images.githubusercontent.com/91045087/232357393-293a7208-be7a-4bf7-b1c4-dfc2c26aa213.png">
I set 30000 iterations for Adam and the L-BFGS. Then I continue training with the RAR method. there are 9600 points in the study area.
And I sampled 6000 points. But it always has over fit problem. How can I fix this problem? 
<img width="794" alt="image" src="https://user-images.githubusercontent.com/91045087/232357754-ffceb43c-b9e1-49bd-ae7a-8fd1c699ff8f.png">

**Comment by tsarikahin** [comment]
I believe this is a bug in DeepXDE. Let me explain. The training data includes both boundary and collocation points. However, test data has only collocation points collected inside the domain (function `tes_points` of `pde.py`). If bc error is large, then you will end up with a larger training error than the test error.  What @lululxvi says about this?

**Comment by lululxvi** [comment]
1. I don't see where is overfitting.
2. It is not a bug. Test data also contains the same boundary points in training.
